### PR TITLE
Revisiting Talking While Coding

### PR DESCRIPTION
**Pull request by zoefarrell**
I think this an hour and a half lesson.... 

I could cut the time for the practice, but I think we want 20 minutes per person so they are not rushed. Thoughts on cutting 30 minutes more into project time? Or we could have this in the morning and not have application work time on Monday?

### CODE REVIEW

**Comment by memcmahon** [comment]
I don't think I want to move this to the morning... I think I would prefer to take from project worktime, if necessary.  I think the job-hunt time is potentially more important 🤷‍♀️   Or, we could ask students to do the review piece as independent pre-work sometime in the morning ?

File: module6/lessons/Week3/RevisitingTalkingWhileCoding.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Warm Up
+
+Think about what you remember about talking while coding (or look back on the initial lesson [here](https://launch.turing.edu/module5/lessons/Week1/TalkingWhileCoding)) and write down your answers to the following questions:
+
+
+1. What sort of things should you say during a technical interview?
+1. What are some benefits of saying these things during a technical challenge?
+
+## Revisiting Four Tip
```
Comment (memcmahon) [inline module6/lessons/Week3/RevisitingTalkingWhileCoding.md]: Should this be Tips?

**Review by memcmahon — APPROVED** [review]

File: module6/lessons/Week3/RevisitingTalkingWhileCoding.md (hunk 1 above)
Comment (zoefarrell) [reply module6/lessons/Week3/RevisitingTalkingWhileCoding.md]: Yep!

File: module6/lessons/Week3/RevisitingTalkingWhileCoding.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+layout: page
+title: Revisiting Talking While Coding
+---
+
+## Learning Goals
+* Revisit the benefits of talking in a technical interview
+* Revisit strategies for talking during a technical interview
+* Strengthen your ability to talk while coding
+
+## Warm Up
```
Comment (zoefarrell) [inline module6/lessons/Week3/RevisitingTalkingWhileCoding.md]: I like the idea of making this Prep and having them work on this sometime in the morning. That way they can take however much time they need. And I don't think that cuts too much into job hunt time.

**Pull request merged**